### PR TITLE
feat(get-started): Add change assets path page

### PR DESCRIPTION
**Pull request by adamledwards**

### CODE REVIEW

**Comment by adamledwards** [comment]
Just saw it regenerated the radio snap. WIll try again

File: src/site/get-started/change-the-image-assets-path.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+If you use a different folder structure than `<YOUR-APP>/assets/images` , you
+can set a Sass variable so that Sass builds the CSS to point to your images
+folder.
+
+Set `$moduk-images-path` before the `@import` line in your Sass file. Set it to
+the path of your images folder. For example:
+
+```bash
```
Comment (jpveooys) [inline src/site/get-started/change-the-image-assets-path.md:16]: Should be `scss`:

```suggestion
```scss
```

File: src/site/get-started/change-the-image-assets-path.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: layouts/markdown-content-page.njk
+title: Change the image assets path
+description: This guide will help you build the CSS to point to your images folder.
+tags: get-started
+order: 300
```
Comment (jpveooys) [inline src/site/get-started/change-the-image-assets-path.md:6]: Seems to be last in the prototype/on Confluene (i.e. `500`) unless you've heard differently.

**Review by jpveooys — COMMENTED** [review]
Nice! Couple of minor comments

**Review by jpveooys — APPROVED** [review]

**Pull request merged**